client/web/src/insights/core/analytics.ts

```diff
@@ -113,44 +110,49 @@ export function getDaysFromInsightStep(step: Duration): number {
 interface InsightGroups {
     codeStatsInsights: number
     searchBasedInsights: number
+    searchBasedBackendInsights: number
+    searchBasedExtensionInsights: number
 }
 
 /**
  * Collect insights count statistic from orgs settings according to
  * auth user organization list.
- * */
-export function getInsightsGroupedByType(
-    settingsCascade: SettingsCascade<Settings>,
-    authUser: AuthenticatedUser
-): InsightGroups {
+ */
+export function getInsightsGroupedByType(settingsCascade: SettingsCascade<Settings>): InsightGroups {
     const { subjects } = settingsCascade
-    const {
-        organizations: { nodes: orgs },
-    } = authUser
-    const orgIDs = new Set(orgs.map(org => org.id))
 
-    const orgSubjects = subjects.filter(subject => orgIDs.has(subject.subject.id))
+    const globalSubjects = subjects.filter(configuredSubject => configuredSubject.subject.__typename === 'Site')
```

Comment:
Is it possible to use subject type-guards here?

```suggestion
    const globalSubjects = subjects.filter(configuredSubject => isGlobalSubject(configuredSubject.subject))
```